.circleci/config.yml

```diff
@@ -1,506 +0,0 @@
-# See https://circleci.com/blog/using-circleci-workflows-to-replicate-docker-hub-automated-builds/
```

Comment:
Should we completely remove the file, or can it be preserved for some more time for legacy reasons (until we are sure the GitHub Action migration has been successful)?

Reply:
I can definitely add it back before our merge to develop. 
However we are just increasing a lot of run on circleCI due to multiple commits being pushed by me, because of that reason I removed the file for now.